sdks/go/pkg/beam/core/runtime/harness/init/init.go

```diff
@@ -43,7 +43,6 @@ var (
 	loggingEndpoint = flag.String("logging_endpoint", "", "Local logging gRPC endpoint (required in worker mode).")
 	controlEndpoint = flag.String("control_endpoint", "", "Local control gRPC endpoint (required in worker mode).")
 	statusEndpoint  = flag.String("status_endpoint", "", "Local status gRPC endpoint (optional in worker mode).")
-	semiPersistDir  = flag.String("semi_persist_dir", "/tmp", "Local semi-persistent directory (optional in worker mode).")
```

Comment:
This is getting passed in through the Boot Container, but that just means we remove it in the boot container, and deprecated/remove the harness flag later.

https://github.com/apache/beam/blob/ff81afdc4adbd02b3686f01fce4160f11bb7246d/sdks/go/container/boot.go#L107

Reply:
And that's probably why the Portable precommit is hanging.

Reply:
Restored flag with a lint:ignore for now. Should turn the portable precommit green